saleor/static/dashboard-next/categories/components/CategoryCreateSubcategories/CategoryCreateSubcategories.tsx

```diff
@@ -0,0 +1,108 @@
+import { withStyles } from "@material-ui/core/styles";
+import * as React from "react";
+import * as classNames from "classnames";
+
+import Card from "@material-ui/core/Card";
+import Button from "@material-ui/core/Button";
+import CardContent from "@material-ui/core/CardContent";
+import Typography from "@material-ui/core/Typography";
+import TextField from "@material-ui/core/TextField";
+import DeleteIcon from "@material-ui/icons/Delete";
+import IconButton from "@material-ui/core/IconButton";
+import AddIcon from "@material-ui/icons/Add";
+// import CardActions from "@material-ui/core/CardActions";
+
+import CardTitle from "../../../components/CardTitle";
+import Toggle from "../../../components/Toggle";
+// import FormSpacer from "../../../components/FormSpacer";
+
+import i18n from "../../../i18n";
+
+interface CategoryCreateSubcategoriesProps {
+  disabled: boolean;
+}
+
+const decorate = withStyles(theme => ({
+  root: {
+    display: "grid",
+    gridColumnGap: theme.spacing.unit * 2 + "px",
+    gridTemplateColumns: "5fr 8fr 0.5fr"
```

Comment:
This doesn't look right. Wouldn't it be better if you use constant width instead of `0.5fr`?